lib/bitvec.cpp

```diff
@@ -137,3 +137,12 @@ unsigned bitvec::ffz(unsigned start) const {
 #endif
     return rv;
 }
+
+bool bitvec::is_contiguous() const {
+    // Empty bitvec is not contiguous
```

Comment:
I think a reasonable argument could be made for returning true if the bitvec is empty, as well. It'd be good to document this method's behavior in the header file.